decidim-core/lib/decidim/core/test/shared_examples/active_support_examples.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+shared_examples "fires an ActiveSupport::Notification event" do |fired_event|
+  it "subscribes to #{fired_event}" do
+    @tested_notification = false
+    subscriber = ActiveSupport::Notifications.subscribe(fired_event) { |_, _| @tested_notification = true }
+
+    command.call
+
+    expect(@tested_notification).to eq(true)
+  rescue RSpec::Expectations::ExpectationNotMetError => e
+    ActiveSupport::Notifications.unsubscribe(subscriber)
+    raise e
```

Comment:
Why do we have this block too if we already have the ensure block below doing the same?

Reply:
I do not remember why i have that particular rescue there ... i can remove it, as the `ensure` part always runs...